routes/areas.js

```diff
@@ -10,6 +10,45 @@ Do not think we need a get areas by year because can just
 use the get all areas request then filter based on year on frontend
 */
 
+// get an area by id
+router.get('/area-management', async (req, res) => {
+  try {
+    const areaInfo = await pool.query(`
+      SELECT area.*, site.num_sites, site.site_info, mt_info.num_mts, student_info.num_students
```

Comment:
Also curious: num_sites, num_mts, and num_students are all returned as strings (it seems like one of the things that COUNT does). If you could return it as an INTEGER I think that may be a bit better for calculation

Reply:
fixed